kubernetes/README.md

```diff
@@ -0,0 +1,31 @@
+# Creating a Kubernetes Deployment
+
+## Summary
+
+The application configuration is in the `config.yaml` Kubernetes secret,
```

Comment:
It's very confusing to name the k8s secret `config.yaml` since that names suggests it is a configmap.  Please rename the file `secret.yaml`. Alternatively, we could put all of the configuration, except for the sensitive values, in a configmap and load that as a volume in the pod. The sensitive values can be loaded as environment variables from a secret.